torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -161,6 +161,48 @@ static inline at::optional<int64_t> parse_as_integer(const std::string& s) {
   return (*str_end == 0) ? at::optional<int64_t>(ans) : at::nullopt;
 }
 
+/*
+Parse default value of IntList declared at native_functions.yaml
+
+There are two kinds of default values:
+1. IntList[2] x=1 (where size=2, value={1,1}
```

Comment:
is it that troublesome to just port everything using style 1 to style 2?  Like, what is the current longest case?

Reply:
I agreed. But I think we are trying to keep both styles.